pages/games/[hash].js

```diff
@@ -97,113 +150,50 @@ function Game({ data, playerHash }) {
                     </div>
                 </div>
             )}
-            <h1 className="text-2xl">
-                Partie <span className="game-hash">{data.game}</span>
-            </h1>
-            {isCurrentPlayer ? (
-                <h2 className="text-green-500 font-bold uppercase">
-                    A moi de jouer
-                </h2>
-            ) : (
-                <h2 className="text-orange-500 font-bold uppercase">
-                    En attente de l'adversaire
-                </h2>
-            )}
-            <div className="flex items-center justify-between my-2">
-                {visibleBoard === PLAYER ? (
-                    <h2>Ma flotte</h2>
+            <div className="flex flex-col items-center p-5">
+                <h1 className="text-2xl">
+                    Partie <span className="game-hash">{data.game}</span>
+                </h1>
+                {isCurrentPlayer ? (
+                    <h2 className="text-green-500 font-bold uppercase">
+                        A vous de jouer !
+                    </h2>
                 ) : (
-                    <h2>Flotte Adverse</h2>
+                    <h2 className="text-orange-500 font-bold uppercase">
+                        En attente de l'adversaire
+                    </h2>
                 )}
+                <div className="relative pb-full w-full max-w-lg">
+                    {displayMessage && (
+                        <div className="absolute z-50 flex w-full h-full items-center justify-center">
+                            <p className="text-4xl">{message}</p>
+                        </div>
+                    )}
+
+                    <div className="absolute grid grid-cols-10 grid-rows-10 w-full h-full gap-px max-w-lg max-h-lg">
+                        {[...Array(100).keys()].map((square, i) => (
```

Comment:
This code should be extracted to a dedicated component in order to avoid generating the squares every time the app is rendered.